e2etest/bgp_test.go

```diff
@@ -465,6 +426,124 @@ var _ = ginkgo.Describe("BGP", func() {
 			),
 		)
 	})
+
+	ginkgo.Context("BFD", func() {
+		table.DescribeTable("should work with the given bfd profile", func(bfd bfdProfile, ipFamily string) {
+			configData := configFile{
+				Pools: []addressPool{
+					{
+						Name:     "bgp-test",
```

Comment:
would be better to rename this something like `bfd-test` (maybe with -bfd.Name suffix)?